build-support/test_first_time_install.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
```

Comment:
We now swallow the output of the Pants script and of `./pants --version` in these tests. Do you all think that's a good thing, or should we add a line to `print(completed_process.stderr)`? 

Note that if the tests fail, then we will output the relevant parts. This is only asking of the value of logging every time so that people can in Travis see what `./pants` ends up logging to the user.

Reply:
After working on followup PRs that build off of this new testing infrastructure—namely converting `ci.py` into `test_sanity_check.py`—I think we should continue to swallow the output *but* always ensure we print the output somehow upon failures. This PR currently does that, so I think we should keep as is.

Reply:
This question is a bit irrelevant actually.

The next followup PR is to convert this test into a Pants target, meaning it will be run through Pytest. Pytest by default silences all stdout and stderr, and only logs if the test fails (https://docs.pytest.org/en/latest/capture.html). This sounds like the ideal behavior. So, in the followup, we can remove the `subprocess.DEVNULL` lines and let the script log normally.

Reply:
+1 to the conclusion here!